sample_projects/LEQuad/main_chibios_sparky.cpp

```diff
@@ -0,0 +1,344 @@
+/*
+    ChibiOS - Copyright (C) 2006..2016 Giovanni Di Sirio
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+#include "boards/sparky_chibi/sparky_chibi.hpp"
+#include "sample_projects/LEQuad/lequad.hpp"
+
+#include "hal/common/time_keeper.hpp"
+
+#include "hal/dummy/serial_dummy.hpp"
+#include "hal/dummy/i2c_dummy.hpp"
+#include "hal/dummy/file_dummy.hpp"
+#include "hal/dummy/adc_dummy.hpp"
+#include "hal/dummy/pwm_dummy.hpp"
+
+#include "simulation/dynamic_model_quad_diag.hpp"
+#include "simulation/simulation.hpp"
+
+#include "drivers/spektrum_satellite.hpp"
+
+#include "drivers/lsm330dlc.hpp"
+#include "drivers/hmc5883l.hpp"
+#include "drivers/gps_ublox.hpp"
+#include "drivers/sonar_i2cxl.hpp"
+
+extern "C"
+{
+    #include "libs/ChibiOS/hal/include/hal_usb.h"
+}
+
+#define MAVLINK_SYS_ID 2
+
+
+int main(void)
+{
+    bool init_success = true;
+
+    // -------------------------------------------------------------------------
+    // Create board
+    // -------------------------------------------------------------------------
+    Sparky_chibi::conf_t board_config = Sparky_chibi::default_config();
+
+    // LEDs on STM32F4DISCOVERY
+    board_config.gpio_led_err =

```

Comment:
Oops this was for tests on the discovery. Done in https://github.com/lis-epfl/MAVRIC_Library/pull/326/commits/e830ba8f343080c191edfa8c9e16f5a8102cd0cd